geoutils/georaster/raster.py

```diff
@@ -527,13 +527,15 @@ def __str__(self) -> str:
 
     def __eq__(self, other: object) -> bool:
         """Check if a Raster's data and georeferencing is equal to another."""
```

Comment:
We should improve the docstring to clearly state what is compared for us to remember.

Reply:
Solved in #317